molgenis-searchall/src/main/java/org/molgenis/searchall/model/EntityTypeResult.java

```diff
@@ -0,0 +1,37 @@
+package org.molgenis.searchall.model;
+
+import com.google.auto.value.AutoValue;
+import org.molgenis.gson.AutoGson;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+@AutoValue
+@AutoGson(autoValueClass = AutoValue_EntityTypeResult.class)
+public abstract class EntityTypeResult
+{
+	public abstract String getId();
+
+	public abstract String getLabel();
+
+	@Nullable
+	public abstract String getDescription();
+
+	public abstract String getPackageId();
+
+	public abstract boolean isLabelMatch();
+
+	public abstract boolean isDescriptionMatch();
+
+	public abstract List<AttributeResult> getAttributes();
+
+	public abstract long nrOfMatchingEntities();
```

Comment:
This method should be getNrOfMatchingEntities. This'll fix the json attribute names as well since then all other get... and is... prefixes will be trimmed.